src/meta/src/stream/stream_manager.rs

```diff
@@ -65,6 +65,8 @@ pub struct CreateStreamingJobContext {
 
     /// DDL definition.
     pub definition: String,
+
+    pub mv_table_id: Option<u32>,
```

Comment:
Should we gradually use `TableID` instead of u32, and maybe in the future, we will extend u32 to u64. Rest LGTM